src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Controls/StringFormatConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Data;
+
+namespace Uno.UI.Tests.Windows_UI_Xaml_Data.xBindTests.Controls
+{
+	public class InvariantStringFormatConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string culture)
+		{
+			if (parameter == null)
+			{
+				parameter = string.Empty;
+			}
+
+			return String.Format(CultureInfo.InvariantCulture, parameter.ToString(), value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'parameter' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436550)